tools/link.py

```diff
@@ -1872,7 +1872,7 @@ def phase_link(linker_arguments, wasm_target, js_syms):
 
 
 @ToolchainProfiler.profile_block('post link')
-def phase_post_link(options, state, in_wasm, wasm_target, target, js_syms, base_metadata=None):
+def phase_post_link(options, state, in_wasm, wasm_target, target, js_syms, base_metadata=None, linker_inputs=None):
```

Comment:
Is this function ever called without the final argument here?

Reply:
Yeah, from `run_post_link`